Tools/glib/dependencies/apt

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
-# If the package $1 is available, prints it. Otherwise prints $2.
+# If the package $1 is available, prints it. Otherwise if package $2 is available, prints $2.
 # Useful for handling when a package is renamed on new versions of Debian/Ubuntu.
 aptIfElse() {
     if apt-cache show $1 &>/dev/null; then
         echo $1
-    else
+    elif apt-cache show $2 &>/dev/null; then
         echo $2
     fi
```

Comment:
While at it, I think it would make sense to produce an error if none of them are available:

```suggestion
    elif apt-cache show $2 &>/dev/null; then
        echo $2
    else
        echo "Neither package is available: $1, $2" 1>&2
        exit 1
    fi
```